typescript/web/src/components/auth-manager/signin-modal/signin-modal.tsx

```diff
@@ -32,25 +32,37 @@ const errors = {
   OAuthCreateAccount: "Try signing in with a different account.",
   EmailCreateAccount: "Try signing in with a different account.",
   Callback: "Try signing in with a different account.",
-  OAuthAccountNotLinked:
-    "To confirm your identity, sign in with the same account you used originally.",
+  OAuthAccountNotLinked: `We could not sign you in. If you have already signed
+  in with this email address, try again using the same method you originally used.
+  Password-less magic link for instance.`,
```

Comment:
Removing the last part to keep it not too long and not too specific.
```suggestion
  OAuthAccountNotLinked: `We could not sign you in. If you have already signed
  in with this email address, try again using the same method you originally used.`,
```

Reply:
Done